src/main/java/at/hannibal2/skyhanni/data/NametagData.kt

```diff
@@ -0,0 +1,25 @@
+package at.hannibal2.skyhanni.data
+
+import at.hannibal2.skyhanni.events.EntityRenderNametagEvent
+import at.hannibal2.skyhanni.test.command.ErrorManager
+import net.minecraft.entity.Entity
+import net.minecraft.util.IChatComponent
+
+object NametagData {
+
+    fun getDisplayName(entity: Entity, name: IChatComponent): IChatComponent {
+        try {
+            if (EntityRenderNametagEvent(entity, name).postAndCatch()) {
```

Comment:
`postAndCatch` already catches all exceptions. Remove this object and put the event directly in the mixin.